stdlib/public/stubs/CMakeLists.txt

```diff
@@ -1,33 +1,40 @@
-set(swift_stubs_objc_sources)
-set(swift_stubs_unicode_normalization_sources)
+set(swift_stubs_sources
```

Comment:
Can you add comments for each group of these sources. Maybe saying something like:

// Platform Independent Sources
set(swift_stubs_platform_independent_sources ...)

// Darwin Only ObjC Sources. This consists of objective-c related stubs.
...

// Sources for Android, Cygwin, FreeBSD, Linux. This consists currently of just ICU sources. [[INSERT EXPLANATION OF WHY WE DO NOT DO THIS ON DARWIN]].
...

I forgot the story with ICU on Darwin, but IIRC @aschwaighofer knows. Arnold, do you remember.